bundles/org.eclipse.jface.text/src/org/eclipse/jface/text/source/inlined/LineContentAnnotation.java

```diff
@@ -158,7 +158,7 @@ StyleRange updateStyle(StyleRange style, FontMetrics fontMetrics, ITextViewer vi
 		return style;
 	}
 
-	static boolean drawRightToPreviousChar(int widgetOffset, StyledText textWidget) {
+	protected boolean drawRightToPreviousChar(int widgetOffset, StyledText textWidget) {
```

Comment:
Since this is new API, this will likely require some `@since 3.26` and a version increase in the MANIFEST.
Or better if you can find a way to not make this method API.

Reply:
thanks a lot for the help. I tried to fix the API tools messages with 80b1736. I added an additional argument to package method LineContentAnnotation#updateStyle . Is this allowed? If not, I would need to introduce method "protected boolean drawRightToPreviousChar" as protected instance method.